packages/editor/src/components/media-placeholder/index.js

```diff
@@ -161,11 +164,53 @@ class MediaPlaceholder extends Component {
 					allowedTypes={ allowedTypes }
 					value={ value.id }
 					render={ ( { open } ) => (
-						<Button isLarge onClick={ open }>
+						<Button
+							isLarge
+							className="editor-media-placeholder__button"
+							onClick={ open }
+						>
 							{ __( 'Media Library' ) }
 						</Button>
 					) }
 				/>
+				{ onSelectURL && (
+					<div
+						className="editor-media-placeholder__url-input-container"
+					>
+						<Button
+							className="editor-media-placeholder__button"
+							onClick={ toggleURLInput }
+							isToggled={ isURLInputVisible }
+							isLarge>
+							{ __( 'Insert from URL' ) }
+						</Button>
+						{ isURLInputVisible && (
+							<URLPopover
+								onClickOutside={ this.closeURLInput }
```

Comment:
@aduth - thanks for catching that. I've switched to onClose. I've undocumented it for URLPopover (which should be fine since URLPopover was only introduced a couple of days ago).